.github/workflows/build.yml

```diff
@@ -0,0 +1,44 @@
+name: Build
+on:
+  schedule:
+    - cron: '0 0 * * *' # Every day at midnight
+  push:
+  pull_request:
+    paths:
+      - '.github/workflows/build.yml'
+      # Build when a fundamental package change has occured
+      - 'package.json'
+
+jobs:
+  build:
+    name: Build the frontend releasable package
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2.1.4
+        with:
+          node-version: '15.x'
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Copy template api keys file
+        run: cp src/constants/apiKeysTemplate.js src/constants/apiKeys.js
```

Comment:
The frontend will be really buggy without real API keys. Is it possible to generate a legit apiKeys.js using GitHub secrets?

Reply:
I'm not sure that's wise. I was hoping to discuss this with you at a later time.

It seems to me that the frontend should default to getting these API keys from houston. In a development environment without a backend, maybe it cascades to checking for a specific file location.

My hope would be that a compiled build can be deployed without any changes. Hence the reason for getting these secret'ish bits from houston. This would also allow all configuration to be done on the backend without doing a redeployment of the frontend, which is always a nice-to-have.

Those are just some thoughts... We can talk about them more later.

Reply:
Hmm, yes, I think when things are fleshed out administrators would enter their API keys as site settings. The one exception to this rule might be the translation service, I need to give that some thought soon..